### PR TITLE
Give more NPC's AI traits

### PR DESCRIPTION
**Pull request by MNG-cataclysm**
#### Summary
Bugfixes "Give proper AI traits to the Isherwood, Kindred, Island Prisoner, Meteorologist, Bunker Merchant, and FOB Arabella NPCs."

#### Purpose of change
Give proper AI traits to more NPCs, as they need them. Continue #62470.

#### Describe the solution
Added proper AI traits to class trait fields. 

#### Describe alternatives you've considered
None.

#### Testing
I went to all NPCs and tested their behaviors. Everything works as intended. 

#### Additional context
I also made some minor edits to the lumbermill employees, added one trait. In addition to AI flags, I also gave the bunker merchant some starting traits for NPCs.

### CODE REVIEW

File: data/json/npcs/isherwood_farm/NPC_Barry_Isherwood.json [hunk 1]
```diff
@@ -17,7 +17,13 @@
     "id": "NC_ISHERWOOD_BARRY",
     "name": { "str": "Barry Isherwood" },
     "job_description": "Missing member of the Isherwood family.",
-    "traits": [ { "group": "NPC_starting_traits" }, { "group": "Appearance_demographics" } ],
+    "traits": [ 
+      { "group": "NPC_starting_traits" }, 
+      { "group": "Appearance_demographics" },
+      { "trait": "IGNORE_SOUND" },
```
Comment (Maleclypse) [inline data/json/npcs/isherwood_farm/NPC_Barry_Isherwood.json:23]: If you give Barry  these traits will it break his quest?  Will he return to the migo cell?  Unless you've tested his interactions he's probably not a good fit for these traits.

**Comment by Maleclypse** [comment]
I'm not sure it makes sense for any of the Isherwood's to have the ignore sound trait.  No bash and return make sense though.

**Comment by Maleclypse** [comment]
A key thing to remember is that these traits are hacks that we would prefer to go away at some point as NPC ai improves and that they should only be added where it's appropriate and not every NPC will be appropriate.

**Review by Maleclypse — APPROVED** [review]
I think this all mostly makes sense now.

File: data/json/npcs/Kindred/NPC_Brigitte_LaCroix.json [hunk 2]
```diff
@@ -12,7 +12,10 @@
       { "trait": "ROBUST" },
       { "trait": "SKIN_DARK" },
       { "trait": "seer_mark" },
-      { "trait": "hair_black_mohawk" }
+      { "trait": "hair_black_mohawk" },
+      { "trait": "IGNORE_SOUND" },
+      { "trait": "NO_BASH" },
+      { "trait": "RETURN_TO_START_POS" }
```
Comment (andrei8l) [inline data/json/npcs/Kindred/NPC_Brigitte_LaCroix.json:18]: You can recruit LaCroix and this makes her stand in place forever

File: data/json/npcs/isherwood_farm/NPC_Chris_Isherwood.json [hunk 3]
```diff
@@ -17,7 +17,12 @@
     "id": "NC_ISHERWOOD_CHRIS",
     "name": { "str": "Chris Isherwood" },
     "job_description": "I'm Jesse and Carlos' son, part of the Isherwood family",
-    "traits": [ { "group": "NPC_starting_traits" }, { "group": "Appearance_demographics" } ],
+    "traits": [
+      { "group": "NPC_starting_traits" },
+      { "group": "Appearance_demographics" },
+      { "trait": "NO_BASH" },
+      { "trait": "RETURN_TO_START_POS" }
```
Comment (andrei8l) [inline data/json/npcs/isherwood_farm/NPC_Chris_Isherwood.json:24]: Same here. In Chris' mission you need to take him to a Mi-Go tower and this prevents that

File: data/json/npcs/other/meteorologist.json [hunk 4]
```diff
@@ -21,7 +21,14 @@
...
     "bonus_per": { "rng": [ -1, 3 ] },
     "worn_override": "NC_meteorologist_worn",
-    "traits": [ { "group": "NPC_starting_traits" }, { "group": "Appearance_demographics" }, { "trait": "BGSS_METEOROLOGIST" } ],
+    "traits": [
+      { "group": "NPC_starting_traits" },
+      { "group": "Appearance_demographics" },
+      { "trait": "BGSS_METEOROLOGIST" },
+      { "trait": "IGNORE_SOUND" },
+      { "trait": "NO_BASH" },
+      { "trait": "RETURN_TO_START_POS" }
```
Comment (andrei8l) [inline data/json/npcs/other/meteorologist.json:30]: Same here. In his final mission, you need to take him to a refugee center

**Review by andrei8l — COMMENTED** [review]
You're applying these traits too broadly. These are just the conflicts I know off the top of my head.

**Comment by MNG-cataclysm** [comment]
> You're applying these traits too broadly. These are just the conflicts I know off the top of my head.

That is fixed now.

**Pull request merged**